source/_integrations/minecraft_server.markdown

```diff
@@ -35,9 +39,14 @@ This integration provides a binary sensor for the following information from a M
 
 This integration provides sensors for the following information from a Minecraft server:
 
-- Latency time
+- Latency
 - Version
 - Protocol version
-- Number of online players (player names are available in state attributes)
+- Number of online players
+  - Including player names list in the state attributes, if available (see note below)
 - Number of maximum players
 - World Message / Message of the Day (MOTD)
+
+<div class='note'>
+Depending on the server, the player names list may not be shown completely. Some servers and plugins are limiting or completely hiding this list or even replacing the player names with fake ones to show some custom message(s) there.
```

Comment:
```suggestion
Depending on the server, the player names list may not be shown completely. Some servers and plugins limit or completely hide this list or even replace the player names with fake ones to show some custom messages there.
```
Tiny style tweak.
We avoid [plural in parentheses](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/s-es)